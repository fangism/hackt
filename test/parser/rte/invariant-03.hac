// "invariant-03.hac"
ebool x, y, a, b;
bool c, d, e, f;
rte {
	x = c;
	y = d;
	a = e;
	b = f;
}
spec {
	$(~(x && y))	// rejects syntax, should be prs-&
}
