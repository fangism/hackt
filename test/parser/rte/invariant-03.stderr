At: "invariant-03.hac":11:
parse error: syntax error
parser stacks:
state	value
#STATE#	(null) 
#STATE#	list<(root_item)>: (instance-decl) ... [2:1--9:1]
#STATE#	keyword: spec [10:1..4]
#STATE#	{ [10:6]
#STATE#	$ [11:2]
#STATE#	( [11:3]
#STATE#	~ [11:4]
#STATE#	( [11:5]
#STATE#	(prs-literal) [11:6]
#STATE#	&& [11:8]
in state #STATE#, possible rules are:
	prs_paren_expr: '(' prs_expr . ')'  (#RULE#)
acceptable tokens are: 
	')' (shift)
